### PR TITLE
Allow pm from room context

### PR DESCRIPTION
**Pull request by bionicpill**
The room context ignores the `to` param so you can't send private messages to users from that context. This should allow Robut to send private messages from room events.

### CODE REVIEW

File: lib/robut/room.rb [hunk 1]
```diff
@@ -25,6 +25,14 @@ def join
 
   # Send +message+ to the room we're currently connected to
   def reply(message, to)
-    muc.send(Jabber::Message.new(muc.room, message))
+    if !to.nil?
```
Comment (justinweiss) [inline lib/robut/room.rb]: Total style nitpick, but would it be better to reverse these clauses?

```ruby
if to.nil?
  ...
else
  ...
```

**Comment by justinweiss** [comment]
:+1:

File: lib/robut/room.rb (hunk 1 above)
Comment (bionicpill) [inline lib/robut/room.rb]: Fixed: 07ba88e2897e0004fb9efd739a18328ecea1b804

**Comment by justinweiss** [comment]
Awesome :+1:

**Pull request merged**